scanpy/tools/_dendrogram.py

```diff
@@ -156,7 +156,10 @@ def dendrogram(
 
     if inplace:
         if key_added is None:
-            key_added = f'dendrogram_{groupby}'
+            if len(groupby) == 1:
+                key_added = f'dendrogram_{groupby[0]}'
+            else:
+                key_added = f'dendrogram_{"_".join(groupby)}'
```

Comment:
This would be just fine I think:

```suggestion
            key_added = f'dendrogram_{"_".join(groupby)}'
```